packages/api-contract/test/contracts_0_96/Erc20.json

```diff
@@ -0,0 +1 @@
+{"name":"Erc20","deploy":{"args":[{"name":"init_value","type":"Balance"}]},"messages":[{"name":"total_supply","selector":949181842,"mutates":false,"args":[],"return_type":"Balance"},{"name":"balance_of","selector":3827153702,"mutates":false,"args":[{"name":"owner","type":"AccountId"}],"return_type":"Balance"},{"name":"allowance","selector":3057219184,"mutates":false,"args":[{"name":"owner","type":"AccountId"},{"name":"spender","type":"AccountId"}],"return_type":"Balance"},{"name":"transfer","selector":3374842663,"mutates":true,"args":[{"name":"to","type":"AccountId"},{"name":"value","type":"Balance"}],"return_type":"bool"},{"name":"approve","selector":431186122,"mutates":true,"args":[{"name":"spender","type":"AccountId"},{"name":"value","type":"Balance"}],"return_type":"bool"},{"name":"transfer_from","selector":197482595,"mutates":true,"args":[{"name":"from","type":"AccountId"},{"name":"to","type":"AccountId"},{"name":"value","type":"Balance"}],"return_type":"bool"}]}
```

Comment:
Ok, don't like the naming here at all. Basically just put it in the contracts folder (unversioned), we only support latest. (I understand it still needs to be updated, but wouldn't want to create a folder that we are removing anyway.) Just replace the files.

Reply:
@amaurymartiny mentioned in another PR that that's the format you agreed on, that's why I used it here. https://github.com/polkadot-js/api/pull/1084#discussion_r302106215

Happy to remove the folder with the older examples here, but what about the general format?

Reply:
General format is fine.

The issue with contracts is that atm, we are certainly not even attempting to support old. So yes, in general the version format is ok, but contracts are still in heavy development even with things like the ABIs, so as soon as we start supporting versions x, y & z it is going to explode.

Reply:
Ok, I removed the old files completely.